code-challenges.test.js

```diff
@@ -59,4 +139,30 @@ var hand3 = [5, 5, 5, 5, 4]
 
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+// declare a function named fullHouse 
+//take variable named count(x) to count the duplicates in array
+//The (for...of) statement iterates over values that the iterable object defines to be iterated over
+//take variable named value count the value in objects
+//conditional statement if/else is used to check the array of values
+
+ const fullHouse = (array) => {
+    let count = {}
+    for(let x of array){
+      count[x] = (count[x] || 0) + 1;
+    }
+    let value = Object.values(count);
+    if((value[0] === 2 && value[1] === 3) || (value[1] === 2 && value[0] === 3)){
+      return true;
+    }else
+    return false;
+}       
+console.log(fullHouse(hand1))
+console.log(fullHouse(hand2))
+console.log(fullHouse(hand3))
```

Comment:
This is a very good solution to this problem that I also got from googling. Your pseudo code is very close to explaining what is happening here and I appreciate that. There is a difference with count being an object and not as array that is useful to dig into if you want to understand what's going on here a bit more. 

I do want to point out that I specifically asked everyone not to rely on console.logs for calling your functions because jest is calling the functions itself. And this suggests to me that you are not using jest the way you will be using it at a job

Reply:
ok sure i will not rely on cosole.logsthank you